src/Log/EventService.php

```diff
@@ -99,23 +100,27 @@ public function populate(?EventEntity $entity): ?AbstractEvent
         $em = $this->em;
 
         $objectClosure = function () use ($em, $objectType, $objectId) {
+            assert($objectType !== null);
```

Comment:
Since this closure is likely called significantly later than this method, a failing assert might be non obvious in the future. For that reason, I'd suggest moving this assert before the closure. This would allow narrowing the type definition of AbstractEvent::entityType as well.